src/controllers/v1/attachment/index.ts

```diff
@@ -336,4 +340,69 @@ export class AttachmentController extends Controller {
     this.memoisedIdentities.set(alias, address)
     this.memoisedIdentities.set(address, alias)
   }
+
+  private async getOidcConfig(oidcConfigUrl: string) {
```

Comment:
Anything to do with getting an attachment from another attachment service should be moved out to a new service class. You'll then be able to unit test that independently of the controller if you want to